modules/matchPattern.js

```diff
@@ -1,21 +1,30 @@
 import pathToRegexp from 'path-to-regexp'
 
 const cache = {}
+const pathToRegexpOptions = {strict: true}
```

Comment:
I would just inline this. No need to extract it out. We're not using it anywhere else and there are only a few potential options anyways.